controllers/petitions.controller.js

```diff
@@ -68,61 +68,75 @@ petitionsController.createWithFund = catchAsync(async (req, res, next) => {
 
 petitionsController.createPetitionWithItems = catchAsync(
   async (req, res, next) => {
-    let { phone, firstName, itemArray, petitionType, description } = req.body;
+    let {
+      phone,
+      firstName,
+      lat,
+      lng,
+      itemArray,
```

Comment:
I commented with the latest code being tested.

Reply:
this one will work with the lastest fe which I had made the pull request. I see you just merge it so please test it again